lib/widgets/layouts/empty_page.dart

```diff
@@ -1,14 +1,17 @@
 import 'dart:math';
 
+import 'package:fluffychat/resource/image_paths.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
 
 class EmptyPage extends StatelessWidget {
```

Comment:
IMO: Change `EmptyPage` => `BlankScreen`

Reply:
also, why it is in the `layouts` directory ?
it should be in the 
- pages/chat_blank/chat_blank.dart
- pages/chat_blank/chat_blank_style.dart